README.md

```diff
@@ -228,3 +228,24 @@ nvm exec 0.10 ./pm2 start servers_extra.json
 
 #### Run
 Once services have started, run `npm start` to open Firefox with a local profile. Access the Hello service as you normally would.
+
+### Running with MailDev
```

Comment:
@vbudhram could you add a ` *******` above the heading to separate from FF Hello